generators/org.eclipse.vorto.codegen.arduino/pom.xml

```diff
@@ -0,0 +1,224 @@
+<project>
```

Comment:
Can you make your generator inherit from the parent, just as the other generators ?